airqo-platform/AirQo-api
# setting up Flask for AI/ML model utilization

- What does this PR do?
This PR updates the ml-service directory so that it can be used to consume ML models and etc. 
- What JIRA task is related to this PR?
There is no JIRA card for this task.
- How do I test out this PR?
Git clone this repo `git pull` and then `cd AirQo-api/src/ml-service` Then run all the necessary set up tasks for Flask apps.
- Which end points should are ready for testing?
There are no endpoints for this PR.
- Quick notes about our Git workflow?
Please refer to [these notes](https://github.com/airqo-platform/AirQo-api/wiki). Special attention should be given to the part of how we create branches and the need to always work outside the **master** branch. And to always ensure that we have the latest master branch code within each branch we are operating it.
